Autocomplete on ENTER works only once

For example, I type 'comm', press ENTER. It executes the command 'commands'.
If I type 'comm' again, press ENTER, I get :
`[ERROR] CommandGroup: _getCommand: Unable to provide with proper autocomplete.`

This goes for any command. Does a first ENTER will work, then any subsequent try gives the error above.

Using Godot 3.1.2

--- a/src/Collection.gd
+++ b/src/Collection.gd
@@ -268,6 +268,7 @@ func size():  # int
 
 # @override  _iter_init(?)
 func _iter_init(arg):  # bool
+	self._iterationCurrent = 0
 	return self._iterationCurrent < self.length
 
 

